ompi/mpi/fortran/use-mpi-f08/Makefile.prototype_files

```diff
@@ -0,0 +1,171 @@
+#
+# Shared list of prototype files to avoid listing dependencies multiple times.
+#
+
+prototype_files = \
```

Comment:
Not sure why this got split out into a separate file...?  I thought the idea was to just use Make transformations rather than re-listing every file.

Reply:
its included in 3 makefiles:
use-mpi-f08/Makefile.am
use-mpi-f08/base/Makefile.am
use-mpi-f08/mod/Makefile.am
I think that's the reason why it was split out into a separate make include file.

Reply:
just to be sure, these .c.in files have nothing to do with the 'c' binding ones other than that they are parsed by the same python infrastructure in the ompi/mpi/bindings folder

Reply:
Got it; makes sense.  Thanks.